src/lib/builders/table-of-contents/create.ts

```diff
@@ -0,0 +1,259 @@
+import { addEventListener, builder, createElHelpers, executeCallbacks, generateId, isBrowser } from "$lib/internal/helpers";
+import type { Defaults } from "$lib/internal/types";
+
+import { onMount } from "svelte";
+import { get, writable, type Writable } from "svelte/store";
+
+import type { Heading, CreateTableOfContentsArgs, ElementHeadingLU, HeadingParentsLU, TableOfContentsItem, NavItemProps } from "./types";
+
+const defaults = {
+    exclude: ['h1'],
```

Comment:
Why exclude h1 by default? Just curious

Reply:
@TGlide The H1 tags are always the article titles so they usually don't appear in a table of contents. Actually that's why I was thinking whether I should remove it all together from the possible values. What do you think?

Reply:
I think giving them the option is nice, sometimes you'd want to include the title. But the defaults make sense